app/(tabs)/_layout.tsx

```diff
@@ -9,22 +9,30 @@ const TabsLayout = () => {
             tabBarActiveTintColor: "#2B4B51"
             }}>
             <Tabs.Screen name="index" options={{
-                headerTitle: "Start",
+                headerTitle: "",
                 headerTitleAlign: "center",
                 headerTintColor: "white",
                 headerStyle: {
-                    backgroundColor: "#2B4B51"
+                    backgroundColor: "#2B4B51",
```

Comment:
There are many instances where you didn't use the ``Colors``  variable, like here for example.